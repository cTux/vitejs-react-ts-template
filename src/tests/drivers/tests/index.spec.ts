import { EmptyTLDriver } from './EmptyTLDriver';

describe('EmptyTLDriver', () => {
  it('should render', async () => {
    const driver = new EmptyTLDriver();
    await driver.render();
    expect(driver.component.innerHTML).toMatchSnapshot();
  });
});
