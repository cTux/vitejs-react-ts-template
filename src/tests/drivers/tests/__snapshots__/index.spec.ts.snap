// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EmptyTLDriver > should render 1`] = `"<div>testing-library-base-driver</div>"`;
